lesson09/index.js

```diff
@@ -18,15 +18,19 @@ function createListDom(itemList) {
 }
 
 function createLoading() {
-    const bodyElement = document.querySelector('.body');
-    console.log(bodyElement)
-    bodyElement.insertAdjacentHTML('afterbegin', '<div class="loading"><img class="loading-img" src="./img/loading-circle.gif"></div></div>');
+    const ulElement = document.getElementById("ul");
+    const loadingImgWrapElement = document.createElement("div");
```

Comment:
IMO：

I think 'element' of '~ element' is Unnecessary. 
Because, clear writing to creating element by `createElement()` methods.
`ulElement` is too I don't think it's necessary 'element' string.  

* ulElement
* loadingImageElement
* anchorElement

Reply:
Thank you for pointing out.
As you say, the constant name becomes unnecessarily long and the role is clear, so I modified the constant name.

870232f